example/90-shoot.yaml

```diff
@@ -354,6 +354,10 @@ spec:
 #   coreDNS:
 #     autoscaling:
 #       mode: horizontal # {horizontal,cluster-proportional}
+#     rewriting:
+#       enabled: true # {true,false}
```

Comment:
This field no longer exist, right?

```suggestion
```

Reply:
Yes, good catch.
Addressed with latest commit.